wire-ios/Wire-iOS/Sources/UserInterface/ConversationList/ListContent/ConversationListContentController/ViewModel/ConversationListViewModel.swift

```diff
@@ -655,7 +655,9 @@ final class ConversationListViewModel: NSObject {
         }
 
         var jsonString: String? {
-            guard let jsonData = try? JSONEncoder().encode(self) else {
+            let encoder = JSONEncoder()
+            encoder.outputFormatting = .sortedKeys
```

Comment:
what's this?

Reply:
We have tests that assert the json string, but the tests are flaky because the order of the keys in the string change. By sorting the keys the tests become stable.